docs/tutorials/typescript/subscriptions/index.md

```diff
@@ -1,416 +1,417 @@
 image: /img/temporal-logo-twitter-card.png
 ---
 
 ![Temporal TypeScript SDK](/img/sdk_banners/banner_typescript.png)
 
-import { OutdatedNotice } from '@site/src/components'
-
-<OutdatedNotice />
-
 ## Introduction
 
-In this tutorial, you will tour all of the [Workflow APIs](https://docs.temporal.io/typescript/workflows#workflow-apis) you should know, primarily Signals, Queries, `condition`, and `sleep` (and eventually Child Workflows and `continueAsNew`), by building a realistic monthly subscription payments workflow that can be canceled and changed while it runs.
-
-- The goal is to give you a more accessible introduction to these APIs by explaining them in context of a realistic application.
-- We also give an example of how you break down project requirements into Activities and Workflow logic.
-
-Your task is to write a Workflow for a limited time Subscription (eg a 36 month Phone plan) that satisfies these conditions:
-
-1. When the user signs up, **send a welcome email** and start a free trial for `TrialPeriod`.
-2. When the `TrialPeriod` expires, start the billing process
-   - If the user cancels during the trial, **send a trial cancellation email**.
-3. Billing Process:
-   - As long as you have not exceeded `MaxBillingPeriods`,
-   - **Charge the customer** for the `BillingPeriodChargeAmount`.
-   - Then wait for the next `BillingPeriod`.
-   - If the customer cancels during a billing period, **send a subscription cancellation email**.
-   - If Subscription has ended normally (exceeded `MaxBillingPeriods` without cancellation), **send a subscription ended email**.
-4. At any point while subscriptions are ongoing, be able to look up and change any customer's:
-   - Amount Charged
-   - Period number (for manual adjustments e.g. refunds)
-
-Of course, this all has to be fault tolerant, scalable to millions of customers, testable, maintainable, and observable.
+In this tutorial, you'll build a subscription application using Temporal and TypeScript. You'll use Temporal Workflows, Activities, sleep, Signals, and Queries to manage the core subscription logic. By leveraging Temporal's capabilities, you can handle complex subscription scenarios directly within Temporal's environment, reducing the need for external databases or queues. This reduces the complexity of the code you have to write and maintain.
 
-:::tip Skip ahead
+You'll create a Temporal Workflow to simulate a subscription process, including free trial periods, billing cycles, and subscription cancellations. This Workflow can be used to query users' billing information, update billing amounts, and cancel users' subscriptions at any time. These interactions will be managed using Temporal's Signals and Queries, enabling real-time updates to ongoing Workflows. You can view the user's entire process through Temporal's Web UI. 
```

Comment:
The functionality of the app would look nice and be easier to consume in a list.